src/scripts.js

```diff
@@ -96,12 +95,6 @@ function checkKeyPressed(event) {
   }
 }
 
-function checkKeyPressedForAdd(event) {
-  if (event.keyCode === 13 && event.target.classList.contains('add-recipe-to-cook-button')) {
-    recipeToCookCard(event)
-  }
-}
-
```

Comment:
Good job determining that this method that was interfering with the enter click to add recipe to cook.